whylog/tests/tests_constraints/test_user_constraint.py

```diff
@@ -34,8 +34,8 @@ def test_verify_fail(self):
 
 class TestTimeConstraint(TestCase):
     def setUp(self):
-        self.min_delta = timedelta(seconds=1)
-        self.max_delta = timedelta(seconds=10)
+        self.min_delta = 1
```

Comment:
Are you sure that we want pass to constraints raw numbers as min_delta? Isn't it better to pass timedelta? If we pass timedelta we don't need to remember units - if they are in seconds or in microseconds. Will it be faster to verify if you pass raw int instead of timedelta?